src/components/accordion/Accordion.tsx

```diff
@@ -116,11 +117,11 @@ export function AccordionHeader({
   children,
   small,
 }: AccordionHeaderProps) {
-  const [isOpen, toggleIsOpen] = React.useState(false);
+  const { isOpen, toggle } = useDisclosure(false);
```

Comment:
`false` is the default value, I think we can omit it.

```suggestion
  const { isOpen, toggle } = useDisclosure();
```